src/btree/bt_curprev.c

```diff
@@ -667,7 +643,11 @@ err:	switch (ret) {
 	case 0:
 		F_SET(cursor, WT_CURSTD_KEY_INT | WT_CURSTD_VALUE_INT);
 #ifdef HAVE_DIAGNOSTIC
-		ret = __wt_cursor_key_order_check(session, cbt, false);
+		/* Skip key order check, if prev is called after a next returned
+		 * a prepare conflict error.
+		 */
+		if (!F_ISSET(cbt, WT_CBT_RETRY_NEXT))
+			ret = __wt_cursor_key_order_check(session, cbt, false);
```

Comment:
Hopefully this is going away (see above), but the first line should be split, that is text only appears on the second line of a block comment.